scss/components/_case-body.scss

```diff
@@ -8,6 +8,29 @@
   }
 }
 
+.civicase__case-body__not-found {
+  background: $crm-white;
+  color: $crm-gray-matte;
+  font-size: 14px;
+  font-weight: $font-weight-heading;
+  height: 100%;
+  padding: 18px 0;
```

Comment:
why are we using this custom padding? It should be one of the following:

* `$padding-large-vertical`: 10px
* `$jumbotron-padding`: 30px
* `$alert-padding`: 15px
* `$panel-body-padding`: 15px